### DISCUSSION TITLE
Linked Breakers?

### DISCUSSION THREAD

**Discussion by cwagz** [jasonacox/Powerwall-Dashboard#141] [General]
I had to change a switch today and got to access the new backup load breaker box that was installed in my garage. All the loads for my house were moved into this box.

I quickly noticed the labeling was totally wrong. I then noticed I now have several linked breakers. These are two adjacent breakers with a bar requiring them to be thrown at the same time. 

I used to be able to shut off parts of a room and my bathroom. Now the entire master bedroom, bathroom and loft all have to be thrown at once. 

Was the installer out of regular breaker?

Is this something required somehow due to my two power walls?

anyone else notice this?

Seemed pretty weird to me honestly. Normally only see this with 220 service or a shared box like the box under your sink that the dishwasher and garbage disposal plug in to.

**Comment by cwagz** [comment]
Thought I would bump this. Just had an issue with my oven. It used to have its own breaker and now it is linked with another appliance. Seems really weird. Anyone else have a ton of linked circuits after getting Powerwalls?

Prior to installation the only linked breakers were the two 220v circuits (A/C) and the unused SPA circuit along with the garbage disposal / dishwasher which share an electrical box so are required by code to turn off together. 

![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/13752647/880f6e7d-9724-44b4-9904-48899d4a8300)

**Reply by jasonacox** [reply]
I agree, it seems off.  During my installation, they did eact one-for-one replacement, where the only linked breakers are for 220v services. Your installation seems like a mistake. You should call them or have an electrician check it out.

Do you have the Tesla plan documents? I had rquested them during build.  I checked mine and it shows single or double pole for each circuit:

<img width="563" alt="image" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/836718/0aef8d18-eb50-44ce-b426-59098ae7e94e">

**Comment by ThePnuts** [comment]
They did similar to me. Before Tesla I had Dryer, Range, Garage on linked breakers

The new sub panel they added, also included: Kitchen GFCI + Disposal, Kitchen GFCI + Micro/hood, 2 sets of Bedroom lights/plugs, 2 other sets of bedroom lights/plugs. They also linked Dryer to my garage GFCI

I did not include my AC under battery backup, which was the cause for the sub panel of the house's load.

**Reply by jasonacox** [reply]
Did they explain why and/or did they fix it?

**Reply by ThePnuts** [reply]
No, but I didn't ask either. My Tesla plan also did not have the detail of the breakers like which was shared above. It just had the major appliances listed and what (the AC) was not going to be on the battery backup. My Uncle is an electrician and when he came to visit several months after install took a short look at everything. He said it looked like the electricians knew what they were doing. He did not look behind the panel cover or anything though.

Only thing from plan document even close.
![image](https://github.com/jasonacox/Powerwall-Dashboard/assets/126296751/3ac68997-4467-4979-a559-6e98b4108803)

Main breaker before moving non-AC
![Old](https://github.com/jasonacox/Powerwall-Dashboard/assets/126296751/8c96f4b7-e7c0-4e22-a920-d8ea06a4f542)

New Sub breaker for battery load. You can see the large number of bridged items.
![New Sub](https://github.com/jasonacox/Powerwall-Dashboard/assets/126296751/e53bc093-2316-4bd9-b876-17d529757ae9)

**Comment by cwagz** [comment]
I am going to try contacting Tesla next week. I would really like it fixed if possible. What is really perplexing is I had another electrician upgrade my main panel about 6 months before the Tesla system was installed. What did they (Tesla) do with all the breakers that were in that panel? It seems like they purposely installed these linked breakers. Both the main panel and this new sub panel use the same Square D stuff. Could it be some way to balance the two Powerwalls? I can't make sense out of that in my head.

**Comment by cwagz** [comment]
I just had a pretty terrible chat session with Tesla. They did send me my plans. The plans show 4 2P breakers and I have 8 2P breakers! The guy basically told me that my system has been installed for 2 years and would not have passed inspection if it did not meet the plan. I laughed at this since the inspector did not even look at anything when he was here. I sent him back a picture of my sub panel and asked why I have 8 2P breakers instead of 4 and he said I don't know and ended the chat.

I also notice they installed the wrong model main disconnect in the sub panel as the opening in the cover does not fit the main disconnect breaker.

I guess I have some work to do and money to spend to solve this one on my own.

**Reply by jasonacox** [reply]
Wow, that's disappointing. It may be worthwhile escalating. It's still under warranty and this should be covered under workmanship defect fixes.

**Comment by cwagz** [comment]
<img width="937" alt="Screenshot 2023-11-14 at 10 05 58 AM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/13752647/037893f7-5be8-468e-b21d-77f91355e872">

**Comment by cwagz** [comment]
This is crazy - I almost looks like my A/C was not even supposed to be moved to the backup load center. There are no 50A breakers shown on the plan. The load calculations in the plan are not even close to what is actually in my backup load panel as far as I can tell. Everything has been working fine but this is really frustrating.

<img width="601" alt="Screenshot 2023-11-14 at 10 15 08 AM" src="https://github.com/jasonacox/Powerwall-Dashboard/assets/13752647/87ddae4c-39ba-4893-950e-e20bba42713b">

**Comment by jgleigh** [comment]
Have you checked that the breakers in question aren't connected to MWBCs (multi-wire branch circuits)? Those are supposed to have a handle-tie, but often don't in older construction. MWBCs share a neutral so you need to turn off both hots at the same time to prevent getting shocked on a neutral you thought was turned off.

**Comment by cwagz** [comment]
@jgleigh - Thank you. I did some reading and I did notice a red wire when replacing the outlets. I bet they are all MWBC. House was built in 2001 and probably did not require tied breakers.

I was thinking the change may have been intentional but just could not figure out why.

On Nov 30, 2023, at 6:27 PM, jgleigh ***@***.***> wrote:

﻿

Have you checked that the breakers in question aren't connected to MWBC (multi-wire branch circuits)? Those are supposed to have a handle-tie, but often don't in older construction. MWBCs share a neutral so you need to turn off both hots at the same time to prevent getting shocked on a neutral you thought was turned off.

**Reply by anderep** [reply]
They always required handle tie. Electricians don’t always follow code.  The wires are also required to be zip tied together to prevent casual separation.
